src/platform/forms-system/src/js/validation.js

```diff
@@ -135,6 +135,7 @@ export function uiSchemaValidate(
   formData,
   path = '',
   currentIndex = null,
+  ancillaryData,
```

Comment:
I'd rather put this right after `formData`, but this is probably where it belongs as far as usage is concerned. What I _really_ wish for is named args, but :man_shrugging: